### DISCUSSION TITLE
I want to be able to OPT-IN on seeing activity from people I do not follow

### DISCUSSION THREAD

**Discussion by Scoppio** [community/community#15206] [Other Feature Feedback, Questions, & Ideas]
Right now there is no way to control whose activity shows up in my activity dashboard, so people that I don't want to contact, the only way to remove their activity from my dashboard is if I block that user, (which also sends me to their profile for some unknown reason!)
In most situations blocking is a too harsh measure and we may just want to NOT SEE THAT PERSONS ACTIVITY.

This is exacerbated by the fact that that person may as well be a bot, which makes it annoying and polutes the activity dashboard with mostly useless activity.

By the way, most of the activities that show in the dashboard is nonsensical and does not add anything of value.

**Comment by corydodt** [comment]
Most of the activity I see is people I "collaborated with", meaning any committer at any past job I've had. I'm glad life goes on for them but I have zero interest in it; worse than useless.
